src/main/java/command/Command.java

```diff
@@ -0,0 +1,219 @@
+package command;
+
+import exception.DukeException;
+import storage.StorageInterface;
+import task.Deadline;
+import task.Event;
+import task.Task;
+import task.Todo;
+import tasklist.TaskList;
+import ui.Ui;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Command class that executes parsed commands given to it.
+ */
+public class Command {
+
+    /** Command portion of the input. */
+    private String commandString;
+    /** Description portion of the input. */
+    private String desc;
+    /** Date when the task starts. */
+    private LocalDate startDate;
```

Comment:
Would it be better if you have subclasses of the Command class that contain the date & time fields? Since commands like "list", "done" and "bye" do not have date and time input.

Reply:
Thanks for the suggestion man. Updated it in my latest commit.